tools/download_ipk.sh

```diff
@@ -41,6 +42,9 @@ main() {
                 usage
                 exit 0
                 ;;
+            -v|--verbose)
+                REDIRECT=""
```

Comment:
```suggestion
                REDIRECT=""
                shift
```

Something tells me you didn't really test this feature :-)